### DISCUSSION TITLE
Does not work with the latest claude API

### DISCUSSION THREAD

**Discussion by ShouweiLi** [danny-avila/LibreChat#3853] [Troubleshooting]
### What happened?

Suddenly does work with the latest Claude API. Here is the error message:
![image](https://github.com/user-attachments/assets/61d9b79b-300b-49ef-a6c2-c567a1a3da16)

### Steps to Reproduce

1. configure Clause API globally
2. send a request from UI
3. get the error.
4. PS: it worked well last week.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It works, I use it constantly.

![image](https://github.com/user-attachments/assets/ecd42a91-2378-4411-9a8b-1c56b06d6a20)

**Reply by danny-avila** [reply]
More info is needed. Are you using the official API? Which model?
